cmd/trovilo/main.go

```diff
@@ -94,6 +95,21 @@ func main() {
 						logEntry.Info("Successfully deleted ConfigMap from namepace")
 					}
 
+					// Deleting the configmap also triggers post-deploy actions
+					if len(job.PostDeploy) > 0 {
+						for _, postDeployAction := range job.PostDeploy {
```

Comment:
We should move this  `for` loop into a dedicated function. It blows up the length of the overall loop. Furthermore we have a similar action in line 162

Reply:
fixed! thx!